### PR TITLE
tests for E-needstest

### PR DESCRIPTION
**Pull request by tamird**
@alexcrichton @jakub-

### CODE REVIEW

File: src/test/run-pass/issue-20225.rs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+#![feature(unboxed_closures)]
+
+struct Foo;
+
+impl<'a, T> Fn<(&'a T,)> for Foo {
+  type Output = ();
+
+  extern "rust-call" fn call(&self, (_,): (T,)) {}
+  //~^ ERROR: has an incompatible type for trait: expected &-ptr
```
Comment (jakub-) [inline src/test/run-pass/issue-20225.rs]: This shouldn't be in `run-pass`, then?

Comment (tamird) [inline src/test/run-pass/issue-20225.rs]: Good catch, thanks. Fixed.

**Comment by jakub-** [comment]
@bors: r+ 13310f2

@tamird Thank you!

**Comment by tamird** [comment]
should be fixed

**Pull request merged**